src/yadism/coefficient_functions/coupling_constants.py

```diff
@@ -46,6 +46,24 @@ def __init__(self, theory_config, obs_config):
         # neutrinos
         for pid in [12, 14, 16]:
             self.weak_isospin_3[pid] = 1 / 2
+
+        # BSM couplings -------------------------------------------------------
+        # Temporary couplings to Olq3
+        self.BSM_coupling_vectorial = {21: 0}
```

Comment:
I wonder is "BSM" too generic? i.e. should we be more specific, like e.g. "4F"*? please tell me! because from my non-existent BSM experience I only know there are many BSM models :upside_down_face: 

*Python variables can not start with a number, but we can solve that problem :upside_down_face:

Reply:
Indeed you are right! I realise I have already started mixing up BSM and 4F labels, 4F is a more precise one and I'll try to stick with it

Reply:
I'd even be more specific and use `Olq3`, as you can have multiple 4F operators right?